### ISSUE TITLE
Bug with 14-5-19 on untimed test 4

### ISSUE THREAD

**Issue by pamelafox** [RunestoneInteractive/RunestoneComponents#577]
This code is claimed to be correct, but it has 2 bugs: 1) it is an infinite loop, since num is never equal to 0, and 2) it has an extra semi-colon after i--

// I.
for (int i = num; num > 0; i--;)
{
    System.out.print (i + " ");
}

**Comment by barbarer** [comment]
You are correct!  I will fix it.  Sorry for the long delay.  I graduated with my PhD in May and changed jobs.  I am now an assistant professor at the University of Michigan
